frontend/pages/passport-application/index.tsx

```diff
@@ -1,3 +1,106 @@
-export default function PassportApplication() {
-  return <div className="passportApplication">I AM PASSPORT!!!!!!</div>
-}
\ No newline at end of file
+import Link from "next/link"
+
+import Logo from "../../shared-components/logo"
+import styles from "./passport.module.css"
+import { USStateSelect } from "../../shared-components/state-select/UsStateSelect"
+
+function handleSubmit() {
+  // TODO: add form submission logic
+}
+
+export default function Passport({ name = "Herbert Q. Placeholder" }) {
+  const {
+    ppWrapper,
+    cityInput,
+    logoWrapper,
+    inputWrapper,
+    ppForm,
+    ppHeading,
+    ppParaWrapper,
+    ppPara,
+    ppRedirect,
+    ppTextarea,
+    ppButton,
+    ppAddress,
+    ppAddInput,
+    ppZip,
+    ppState,
+    ppCity,
+    zipInput,
+    ppLabel
```

Comment:
Not a blocker, but if you wanted to reduce the length here you could utilize CSS nesting to not need quite so many classNames from the module!